crates/sncast/tests/e2e/account/delete.rs

```diff
@@ -65,7 +65,7 @@ pub async fn test_delete_abort() {
         "--name",
         "user3",
         "--network",
-        "alpha-goerli2",
+        "alpha-goerli",
```

Comment:
This value is not derived from the accounts file. Instead, it's created in the `create_tempdir_with_accounts_file` function. Essentially, it could be any value at this point.